src/app/cpu-utilization-monitor/cpu-observer-list/cpu-observer-list.component.ts

```diff
@@ -31,8 +31,9 @@ export class CpuObserverListComponent {
     this.endpointService.addEndpoint(endpoint);
   }
   /**
-   * open dialog and then adding endpoint after the dialog closes
-   * but only if the res is not undefined
+   * open dialog and add endpoint after the dialog closes
+   *
+   * only add endpoint if the res is not defined
```

Comment:
endpoint hinzufügen, genau dann wenn es defined ist, oder? 
```suggestion
   * only add endpoint if the res is defined
```

Reply:
Ja habe ich korrigiert